code/modules/cm_marines/equipment/kit_boxes.dm

```diff
@@ -64,6 +65,12 @@
 	new /obj/item/ammo_magazine/pistol/vp78(src)
 	new /obj/item/weapon/gun/rifle/sniper/M42A(src)
 	new /obj/item/facepaint/sniper(src)
+	// spotter
+	new /obj/item/clothing/head/helmet/marine/ghillie(src)
+	new /obj/item/clothing/suit/storage/marine/ghillie(src)
+	new /obj/item/storage/backpack/marine/smock(src)
+	new /obj/item/device/binoculars/range/designator/spotter(src)
+	new /obj/item/pamphlet/skill/spotter(src)
```

Comment:
consider making another kit-in-a-kit to streamline actually giving this to the spotter

or just make the kit dispense 2 kits - the spotter and main sniper